### DISCUSSION TITLE
API Integrations - Details and Placements

### DISCUSSION THREAD

**Discussion by SanjanaIL** [ONDC-Official/ONDC-FIS-Specifications#43] [General]
![image](https://github.com/ONDC-Official/ONDC-FIS-Specifications/assets/142986416/b5070bdc-7981-407b-94ff-0611b21c36a0)
1) Is this flow set in order and will the implementation be done in the same sequence?
2) Can we please get a documentation for these APIs to understand the request response param and an idea on which part of the flow will each of these APIs be implemented

**Comment by 92shreyansh** [comment]
API documentation is present in the Swagger UI. These are the requence/flow of the transaction where in each step is a API or form illustration
